src/components/common/icons/Facebook.jsx

```diff
@@ -0,0 +1,20 @@
+const FacebookIcon = () => {
```

Comment:
https://www.freecodecamp.org/news/how-to-import-svgs-in-react-and-vite/

Let's use 6th approach to handle svgs